sources/packages/web/src/services/ApplicationService.ts

```diff
@@ -11,7 +16,42 @@ export class ApplicationService {
   public async getNOA(applicationId: number): Promise<any> {
     return ApiClient.Application.getNOA(applicationId);
   }
+
+  public async getApplicationData(applicationId: number): Promise<any> {
```

Comment:
I have also added this service with an interface, in my current PR, we can have a discussion. was there any reason to use `any`.?

Reply:
This method is marked as changed because it was probably moved, but it was not introduced by this PR. There is no reason to be any. Your change should be the one to be persisted, but this change is not part of this PR.